### ISSUE TITLE
[suggestion] add --quiet option for windows conan install

### ISSUE THREAD

**Issue by Belpaire** [conan-io/conan#7857]
When I go to https://conan.io/downloads.html and choose the windows conan install, it seems it does not have a --quiet install option.

Why this would be handy to us:

On  our windows installations we prefer using py instead of python on the commandline/scripts. Usually when installing any version of python on windows, it will add the py.exe on the windows folder, (it is an option in the python installation). 

This way we don't need to worry about python being on the path, as py.exe is responsible for choosing the right python installation. It gives developers the freedom of not being bound to specific versions of python. However this also means that we cannot rely on the fact that the command: 'py -m pip install conan', will make conan available on the path. Thus I tried looking into if I could install a standalone conan and either put it on a directory I know will be on the path, or to put it in a directory which will be the same over all development systems. 

Having a --quiet install would make adding it to the installation scripts quite easy, but as it seems it is not possible at the moment I cannot go this route. I don't know how easy it is for you to add this functionality or to give me a workaround?

**Comment by memsharded** [comment]
Hi @Belpaire 

The Windows installer is created with InnoSetup, have you tried the ``/SILENT, /VERYSILENT`` comand line arguments?

The other issues, I don't fully understand, as for the installers, the python interpreter is fully bundled, it has nothing to do with the system python, it doesn't even need it. Could you please clarify this point?

**Comment by Belpaire** [comment]
Hi @memsharded , 

Must have missed the /SILENT, I had tried --help to see other options but had forgotten to test /? which does give info. As for the other info, it wasn't specific to conan not working or anything, but just why I wanted to use the standalone installer instead of any system python. It's more of a context I tried to give. But with the /SILENT I can get on (although I find it strange that /SILENT installs the x64 conan to x86 program files). Edit: accidentally used x86 installer, ignore this comment.

Kind regards.
